.circleci/print_license.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Why do we need both Bash and Ruby for this? Seems like this could be done using only 1 language 🤷